src/Admin/SiteHealth.php

```diff
@@ -0,0 +1,386 @@
+				]
+			);
+		}
+
+		return array_merge(
+			$data,
+			[
+				'status' => 'good',
+				/* translators: %d the ICU version */
+				'label'  => sprintf( esc_html__( 'The version of ICU, %d, looks good.', 'amp' ), $icu_version ),
+			]
+		);
+	}
+
+	/**
+	 * Adds debug information for AMP.
+	 *
+	 * @param array $debugging_information The debugging information from Core.
+	 * @return array The debugging information, with added information for AMP.
+	 */
+	public function add_debug_information( $debugging_information ) {
+		return array_merge(
+			$debugging_information,
+			[
+				'amp' => [
+					'label'       => esc_html__( 'AMP', 'amp' ),
+					'description' => esc_html__( 'Debugging information for the Official AMP Plugin for WordPress.', 'amp' ),
+					'fields'      => [
+						'amp_mode_enabled'        => [
+							'label'   => 'AMP mode enabled',
+							'value'   => AMP_Theme_Support::get_support_mode(),
+							'private' => false,
+						],
+						'amp_experiences_enabled' => [
+							'label'   => esc_html__( 'AMP experiences enabled', 'amp' ),
+							'value'   => $this->get_experiences_enabled(),
+							'private' => false,
+						],
+						'amp_templates_enabled'   => [
+							'label'   => esc_html__( 'Templates enabled', 'amp' ),
+							'value'   => $this->get_supported_templates(),
+							'private' => false,
+						],
+						'amp_serve_all_templates' => [
+							'label'   => esc_html__( 'Serve all templates as AMP?', 'amp' ),
+							'value'   => $this->get_serve_all_templates(),
+							'private' => false,
+						],
+					],
+				],
+			]
+		);
+	}
+
+	/**
+	 * Gets the AMP experiences that are enabled.
+	 *
+	 * @return string The experiences, in a comma-separated string.
+	 */
+	public function get_experiences_enabled() {
+		$experiences = AMP_Options_Manager::get_option( 'experiences' );
+		if ( empty( $experiences ) ) {
+			return esc_html__( 'No experience enabled', 'amp' );
+		}
+
+		return implode( ', ', $experiences );
+	}
+
+	/**
+	 * Gets the templates that support AMP.
+	 *
+	 * @return string The supported template(s), in a comma-separated string.
+	 */
+	public function get_supported_templates() {
+		$possible_post_types = AMP_Options_Manager::get_option( 'supported_post_types' );
+
+		// Get the supported content types, like 'post'.
+		$supported_templates = array_filter(
+			AMP_Post_Type_Support::get_eligible_post_types(),
+			static function( $template ) use ( $possible_post_types ) {
+				$post_type = get_post_type_object( $template );
+				return (
+					post_type_supports( $post_type->name, AMP_Post_Type_Support::SLUG )
+					||
+					( ! AMP_Options_Manager::is_website_experience_enabled() && in_array( $post_type->name, $possible_post_types, true ) )
+				);
+			}
+		);
+
+		// Add the supported templates, like 'is_author', if not in 'Reader' mode.
+		if ( AMP_Theme_Support::READER_MODE_SLUG !== AMP_Theme_Support::get_support_mode() ) {
+			$supported_templates = array_merge(
```

Comment:
These are not translated, as this is debugging information that we might get on a WordPress.org support topic. It could be confusing to see these in different languages.